ppo.py

```diff
@@ -21,46 +21,48 @@
 
 peft_config = LoraConfig(
     lora_alpha=16,
-    #lora_dropout=0.1,
+    # lora_dropout=0.1,
     r=16,
     bias="none",
     task_type="CAUSAL_LM",
 )
 
-base_model, model, tokenizer = llm_config.load_model()
+base_model, model, tokenizer = hugginface_config.load_model()
 base_model.config.use_cache = False
 # More info: https://github.com/huggingface/transformers/pull/24906
 base_model.config.pretraining_tp = 1
 
-if ppo_model_path is None:
+if PPO_MODEL_PATH is None:
     model = AutoModelForCausalLMWithValueHead.from_pretrained(
-        model,
-        trust_remote_code=True,
-        device_map="auto",
-        peft_config=peft_config)
+        model, trust_remote_code=True, device_map="auto", peft_config=peft_config
+    )
 
 ppo_trainer = PPOTrainer(
     model=model,
     config=config,
     tokenizer=tokenizer,
 )
 
-model_generation_args = llm_config.get_model_generation_args(tokenizer)
+
+model_generation_args = huggingface_generate.get_model_generation_args(tokenizer)
 model_generation_args["min_length"] = -1
 model_generation_args["top_k"] = 0.0
 model_generation_args["top_p"] = 1.0
 
-def generate(prompt):
+
+def generate(prompt: str) -> (str, (List[str], List[str]):
```

Comment:
I don't think this type is right. (qs, ts) are tensors not list of strings?

Reply:
Removed.